modules/http4s-client/src/main/scala/trace4cats/http4s/client/ClientSyntax.scala

```diff
@@ -1,28 +1,90 @@
 package trace4cats.http4s.client
 
 import cats.effect.kernel.MonadCancelThrow
+import org.http4s.{Headers, Request, Response}
 import org.http4s.client.Client
+import org.typelevel.ci.CIString
 import trace4cats.context.Provide
 import trace4cats.http4s.common.Http4sSpanNamer
-import trace4cats.model.TraceHeaders
+import trace4cats.model.{AttributeValue, TraceHeaders}
 import trace4cats.optics.{Getter, Lens}
 import trace4cats.{Span, ToHeaders}
 
 trait ClientSyntax {
   implicit class TracedClient[F[_]](client: Client[F]) {
+
+    @deprecated("Use liftTraceExtended instead", "0.14.1")
```

Comment:
Because Scala doesn't allow overloading methods with default arguments, bincompat issue couldn't be solved here by making this method package private and overloading with the new method. Hence, this method is deprecated and the newer method had a different name. This also comes with the cost that people will be forced to change to the new method anyways. The alternate (maybe better) option here would be to introduce a breaking change.

Reply:
Given that trace4cats-kafka recently went with a breaking change and a release of the http4s integration should also include the open PR and hence the changes coming from CE 3.5, I think we can go for a breaking change and keep the code clean. If bincompat was taken seriously, we would require MiMa setup anyway.

Reply:
That makes sense, let's keep the code clean and go for a breaking change then.